ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -267,6 +267,28 @@ Listen <%= addr_port %>
     <%- end -%>
   </Location>
 
+  <%- if @api_cors_enabled -%>
+  # CORS setup for API requests
+  <Location "<%= @api_cors_uri %>">
+    Header always set Access-Control-Allow-Origin "<%= @api_cors_origin %>"
+    Header always set Access-Control-Allow-Methods "POST, GET, OPTIONS, DELETE, PUT"
+    Header always set Access-Control-Allow-Headers "x-requested-with, content-type, origin, authorization, accept, client-security-token"
+    Header always set Access-Control-Expose-Headers "Content-Security-Policy, Location"
+    Header always set Access-Control-Allow-Credentials "false"
+    Header always set Access-Control-Max-Age "600"
+
+    # CORS PREFLIGHT REQUESTS ARE NOT AUTHENTICATED.
+    <Limit OPTIONS>
+      allow from all
+      Satisfy Any
+    </Limit>
+
+    RewriteEngine On
+    RewriteCond %{REQUEST_METHOD} OPTIONS
+    RewriteRule ^(.*)$ $1 [R=204,L]
+  </Location>
+  <%- end -%>
+
```

Comment:
I believe this extra newline is causing unit tests to fail because the previously expected rendered configs are now different due to a new line.  Either the unit tests need to be updated or move the `<%- end -%>` down one line to include the newline. The unit tests are sort of tricky to update as part of them also compares the SHA checksum of rendered configs so if the configs change you must determine new SHA sum.

Reply:
Those steps should work to test from `ood-portal-generator` directory.  There are a bunch of extra steps to take if you run those commands from root of repo, but much simpler if you run from `ood-portal-generator` directory.

Reply:
Thanks Trey,

I was able to run the tests locally from the `ood-portal-generator` directory.
Removing the space fixed the tests.

I have updated the "generates a template with all configurations supplied" to include the CORS section